extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/sun.xml

```diff
@@ -30,27 +30,27 @@
 				<description>The morning night event range</description>
 			</channel-group>
 			<channel-group id="astroDawn" typeId="sunRange">
-				<label>Astro Dawn</label>
```

Comment:
We actually have the (undocumented...) best practice to keep labels always capitalised - so this one was actually fine.

Reply:
No problem. Then I will do it the other way around. Revert these changes and change the ones which do not fit to the new pattern.

Is this true for the translated labels too?

Reply:
Yes, imho makes sense to have a common way in the UI.
Should definitely also be documented, but haven't found a good place and the time yet :-/